### PR TITLE
valentine's day food

### PR DESCRIPTION
**Pull request by robintime**
vdrink
vfood
vcoo

:cl:
 * tweak: holiday cookies crumb now
 * rscadd: adds valentine's cookies to the holiday cookie selection
 * rscadd: adds the pink lady cocktail, made with 3 parts gin and 1 part cherry jelly
 * rscadd: adds candy hearts, which all say different things upon examine, and come in different colors.
 * rscadd: adds chocolate strawberries, which are made with berries and a chocolate bar

### CODE REVIEW

**Comment by d3athrow** [comment]
👎 not pigeoms

**Comment by Eneocho** [comment]
How many candy hearts are made per cooking cycle?

**Comment by robintime** [comment]
> How many candy hearts are made per cooking cycle?

three

**Comment by Eneocho** [comment]
> > How many candy hearts are made per cooking cycle?
> 
> three

This is nitpicky autism, but you lose a ton of nutrition on the recipe. You have 105 nutrition before (100 from oil, 5 from the sugar), you get 21x3 after (15 from nutriment, 6 from sugar) for a total of 63 nutrition.
I'd either add 1u extra nutriment and remove 1u sugar (35 nutrition, 30 from nutriment, 5 from sugar, for a total of 105) or change the recipe to make 2 more candy (for exactly 105 nutrition as well).

**Comment by SonixApache** [comment]
here's some ideaguy power: add some sort of heart-shaped condiment for overlays

**Comment by robintime** [comment]
> This is nitpicky autism, but you lose a ton of nutrition on the recipe. You have 105 nutrition before (100 from oil, 5 from the sugar), you get 21x3 after (15 from nutriment, 6 from sugar) for a total of 63 nutrition. I'd either add 1u extra nutriment and remove 1u sugar (35 nutrition, 30 from nutriment, 5 from sugar, for a total of 105) or change the recipe to make 2 more candy (for exactly 105 nutrition as well).

this might not be entirely accurate because iirc the way multispawners do reagents is fucking weird  and i cannot into math but this is a good point, i'll make these changes tomorrow after a bit of testing



> here's some ideaguy power: add some sort of heart-shaped condiment for overlays

can you elaborate?

**Comment by SonixApache** [comment]
you know how mustard and ketchup and sugar and hot/coldsauce and whatnot add specific overlays when added to food? 
make some condiment that does that too but in the shape of a heart

maybe just use the pink lady cocktail for it

**Comment by Eneocho** [comment]
Condiments can make overlays on food, think of it as a proto-painting system (a guy made a canadian flag with ketchup and mayo, red and white). He's suggesting that you make a special chem with it's own heart shaped overlay for muh fluff.

**Comment by robintime** [comment]
> maybe just use the pink lady cocktail for it

you know, i think i have just the thing, but it most likely won't be in this PR; i am currently reviving my old roses project in time for valentines day, and rose oil (which you get from grinding rose petals) could do this very nicely. the only thing really bottlenecking it is that i am not a spriter and don't want to sprite the in-trays, and i have never grown roses so im not sure what their botany stats should be.



<s>also, just as a reminder to myself: i need to make the rind garnish on the pink lady sprite more clear that it's a thin spiraling strip...</s> done

**Comment by Eneocho** [comment]
You can just look for some rose bud and rose bush pictures for ideas.
Bush for baby stage. 
Bush with green buds and bush with red-green buds for growing stages. 
Bush with harvestable roses for the adult stage. 
Dead bush with brown roses for the dead stage.

**Comment by Kurfursten** [comment]
> the way multispawners do reagents is fucking weird and i cannot into math but this is a good point, i'll make these changes tomorrow after a bit of testing

Worst case you can just cook one and VV to see the reagents in one

**Comment by Eneocho** [comment]
Now I don't get it. Do the candy have 3 nutriment and 15 sugar each? Or is it distributed to the three? If it's the latter, add 3 nutriment so each candy gets 2 nutriment and the total nutriment is equal to the ingredients.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -668,14 +670,55 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/candyheart
+	name = "candyheart"
+	icon = 'icons/obj/food.dmi'
+
+/obj/item/weapon/reagent_containers/food/snacks/candyheart/New()
+	..()
+
+	var/heartphrase = null
+	if(!heartphrase)
+		heartphrase = pick( list("SO FINE","B TRU","U ROCK","HELLO","SOUL MATE","ME + U","2 CUTE","SWEET LUV","IM URS","XOXO","B MINE","LUV BUG","I &lt;3 U","PDA ME") )
```
Comment (DamianX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]:
```suggestion
	var/heartphrase = pick("SO FINE","B TRU","U ROCK","HELLO","SOUL MATE","ME + U","2 CUTE","SWEET LUV","IM URS","XOXO","B MINE","LUV BUG","I &lt;3 U","PDA ME")
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -668,14 +670,55 @@
...
+
+	var/heartcolor = null
+	if(!heartcolor)
+		heartcolor = pick( list("p","b","w","y","g") )
```
Comment (DamianX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]:
```suggestion
	var/heartcolor = pick("p","b","w","y","g")
```

**Pull request merged**